lib/render/RayCaster.cpp

```diff
@@ -745,6 +927,16 @@ int RayCaster::_initializeFramebufferTextures()
     glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
     glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_WRAP_R, GL_CLAMP_TO_EDGE);
 
+    /* Generate and configure 3D texture: _2ndVarDataTexId */
+    glGenTextures( 1, &_2ndVarDataTexId );
```

Comment:
All these textures have the same parameters so you could consolidate their initialization

Reply:
This is addressed in eedfe73.